.github/workflows/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
+
```

Comment:
- **[OPTIONAL]** It is a good practice to have a PR summary that describes your achievements or changes you've committed so far, I highly suggest this good practice 🎯.
![image](https://user-images.githubusercontent.com/68030297/219417145-5b4d76a1-7271-4261-a6ea-0d0d693b6d8f.png)